### ISSUE TITLE
fiber number dependences of labels - slightly improved when fiber number added as a label

### ISSUE THREAD

**Issue by mkness** [mkness/TheCannon#51]
There is a trend seen with fiber number in the labels returned, likely due to the different LSF. Adding fiber number as a training label improves the offsets but does not fix this problem entirely - also fails on some stars entirely in the training step run through the test step. The figure with 5 panels below shows the tends seen and the figure with 6 panels is when fiber number is added as an additional label. 
![labels_and_fiber_no](https://cloud.githubusercontent.com/assets/937118/8355175/f1ce0d4c-1b4d-11e5-8752-ff717029c262.png)
![labels_and_fiber_no_trainedonfib](https://cloud.githubusercontent.com/assets/937118/8355179/f61a5e78-1b4d-11e5-82c2-7a7bc73fbaf8.png)

**Comment by mkness** [comment]
Now add fiber number not as additional label but as additional data point for training and test stars.

**Comment by davidwhogg** [comment]
Woah crazy talk!  I don't think this is quite the right approach, even though it is very promising here!!  I think the right approach is to explicitly deal with resolution (since we know what fiber number does).  But let's put this on the back burner and discuss in HD when I get there.  Unless you are *stoked*.

**Comment by HWRix** [comment]
I agree that dealing explicitly with resolution is superior. But the fiber number is a datum both in the training and the test step; so conceptually treating it as a data point seems OK.
